demo/pkg/subgraphs/courses/patches/grpc-health-check@2.1.0.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/build/src/health.js b/build/src/health.js
```

Comment:
We need to somehow document this. Having an explanation only in the PR description will eventually cause us to forget why this needs to be patched. This is a very hacky workaround

Reply:
I added a comment in the cli in init.ts, will also add in documentation

I removed the fs-polyfill.ts (was not working properly when testing) and replaced it with a patch for [this](https://github.com/protobufjs/protobuf.js/blob/master/lib/inquire/index.js), found out that there build step is broken. Last update 9 years ago: https://www.npmjs.com/package/@protobufjs/inquire/v/1.1.0, related complaints: https://github.com/protobufjs/protobuf.js/issues/2099